markup.go

```diff
@@ -166,7 +166,10 @@ func Class(class ...string) Applyer {
 func mustValidateClassNames(class []string) {
 	for _, name := range class {
 		if containsSpace(name) {
-			failClassName(name)
+			panic(
+				`vecty: invalid class string "` + name + `" with spaces passed to ` + "`vecty.Class` -- " +
+					`you must pass each name as a separate argument`,
+			)
```

Comment:
Sorry @marwan-at-work I should've been more clear. I actually want this panic message to just state:

1. The argument to `vecty.Class` is invalid.
2. The string argument cannot contain a space.

And nothing more. That is why I suggested just writing:

```Go
panic(`vecty: invalid argument to vecty.Class "` + name + `" (string may not contain spaces)`)
```

here instead of the `failClassName` call. I didn't mean to just inline it, but rather to _also_ make the panic message more brief.

I don't think that we should advise the user on what to do (`you must pass each name as a separate argument`) but rather just tell them what they did wrong. This is more consistent with other panics inside Vecty, where it is highly impossible for us to tell the user what they should change in their code. So I think it's bad to tell the user how to fix their code here.

Reply:
If we decide that more context on how to resolve various vecty errors is required, we can implement error codes or something and maintain documentation in the wiki.

Reply:
@slimsag not telling them how to fix their code, but telling them exactly what is wrong with their code makes a lot of sense :)